<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasveer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tasveer.qrc">
    <normaloff>:/about/images/icon.ico</normaloff>:/about/images/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="leftFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="searchFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="methodBox">
             <property name="toolTip">
              <string>Method for tag filtering</string>
             </property>
             <item>
              <property name="text">
               <string>OR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AND</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tagSearchBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="tagsList"/>
        </item>
        <item>
         <widget class="QFrame" name="filterFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="filterMethodBox">
             <property name="toolTip">
              <string>Method for image filtering</string>
             </property>
             <item>
              <property name="text">
               <string>OR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AND</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="filterButton">
             <property name="text">
              <string>Filter Images</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="rightFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="dirSelectBox">
          <property name="toolTip">
           <string>Filter images by directories</string>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imageSearchBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="showUntaggedCheckbox">
          <property name="text">
           <string>Show only untagged images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="images"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddDirectory"/>
    <addaction name="actionAddImage"/>
    <addaction name="actionAddTags"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddDirectory">
   <property name="text">
    <string>Add Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAddImage">
   <property name="text">
    <string>Add Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAddTags">
   <property name="text">
    <string>Add Tags</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../tasveer.qrc"/>
 </resources>
 <connections/>
</ui>
